tests/app/commands/test_populate_redis.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime
+
+from freezegun import freeze_time
+import pytest
+
+from app.commands import populate_redis_template_usage
+
+from tests.conftest import set_config
+from tests.app.db import create_notification, create_template, create_service
+
+
+def test_populate_redis_template_usage_does_nothing_if_redis_disabled(mocker, notify_api, sample_service):
+    mock_redis = mocker.patch('app.commands.redis_store')
+    with set_config(notify_api, 'REDIS_ENABLED', False):
+        with pytest.raises(AssertionError):
```

Comment:
Should we be throwing an exception here. 

Redis being turned off is not exceptional circumstances? If it is RedisConfigError?

Reply:
it's turned off locally.